sdks/python/apache_beam/runners/pipeline_context.py

```diff
@@ -124,6 +126,13 @@ def __init__(self, proto=None, default_environment=None):
           Environment(default_environment))
     else:
       self._default_environment_id = None
+    self.use_fake_coders = use_fake_coders
```

Comment:
Can we add a brief description of this feature, and if this is meant to be temporary, link to a JIRA to resolve this workaround?

Reply:
Done.